packages/circuit-ui/components/Popover/Popover.spec.tsx

```diff
@@ -32,7 +32,7 @@ import {
 
 jest.mock('@sumup/collector');
 
-const placements: Placement[] = ['auto', 'top', 'bottom', 'left', 'right'];
+const placements: Placement[] = ['top', 'bottom', 'left', 'right'];
```

Comment:
Did Popper.js also used to accept an `auto` prop? What did it do?

Removing support for it would also be a breaking change (imagine if a developer was using `<Popover placement="auto" />` before; now it might break). It's fine but we should take this decision explicitly (and add it to the changesets)

Reply:
Awesome, thanks for looking into this and for the detailed reply! I'm also tending towards not exposing `auto`—as far as I know it wasn't being used much, and we have the fallback placements logic built in anyways. Let's keep the API simple here and remove it 🙂